src/components/Authentication/Login.vue

```diff
@@ -90,6 +91,10 @@ export default {
       type: Array,
       default: () => [],
     },
+    showForgotPassword: {
+      type: Boolean,
+      default: true,
```

Comment:
Per #123 we sort of established a negative default for props.  Could change this to `hideForgotPassword` or simply change the default to false.

I know there was another discussion about having strict false for boolean props, but I don't remember where it went...

Reply:
Sure thing